kanidmd/src/lib/actors/v1_write.rs

```diff
@@ -765,7 +758,7 @@ impl Handler<PurgeTombstoneEvent> for QueryServerWriteV1 {
 
     fn handle(&mut self, msg: PurgeTombstoneEvent, _: &mut Self::Context) -> Self::Result {
```

Comment:
It looks like you've removed the result here?

Reply:
Good one. Here clippy complains about that we assign to `res` the unit value. `type Result = ()` is defined to this trait, so the only value that can be returned is `()` for `handle` IIUC.